traffic_ops/traffic_ops_golang/cdni/shared.go

```diff
@@ -738,32 +802,48 @@ const (
 	CountryCode               = "countrycode"
 )
 
+// GenericHostMetadata contains the generic CDNi metadata for a requested update to a specific host.
 type GenericHostMetadata struct {
 	Host         string           `json:"host"`
 	HostMetadata HostMetadataList `json:"host-metadata"`
 }
 
+// GenericRequestMetadata contains the generic CDNi metadata for a requested update.
 type GenericRequestMetadata struct {
 	Type     string          `json:"type"`
 	Metadata json.RawMessage `json:"metadata"`
 	Host     string          `json:"host,omitempty"`
 }
 
+// HostMetadataList contains CDNi metadata for a specific host.
 type HostMetadataList struct {
 	Metadata json.RawMessage `json:"metadata"`
 }
 
+// GenericMetadata contains generic CDNi metadata.
 type GenericMetadata struct {
 	Type  SupportedGenericMetadataType `json:"generic-metadata-type"`
 	Value json.RawMessage              `json:"generic-metadata-value"`
 }
 
+// CapacityRequestedLimits contains the requested capacity limits.
 type CapacityRequestedLimits struct {
 	RequestedLimits []CapacityLimit `json:"requested-limits"`
 }
 
+// CapacityLimit contains the limit information for a given footprint.
 type CapacityLimit struct {
 	LimitType  string      `json:"limit-type"`
 	LimitValue int64       `json:"limit-value"`
 	Footprints []Footprint `json:"footprints"`
 }
+
+// ConfigurationUpdateRequest contains information about a requested CDNi configuration update request.
+type ConfigurationUpdateRequest struct {
+	ID            int             `json:"id"`
+	UCDN          string          `json:"ucdn"`
+	Data          json.RawMessage `json:"data"`
+	Host          string          `json:"host"`
+	RequestType   string          `json:"request-type" db:"request_type"`
+	AsyncStatusID int             `json:"async-status-id" db:"async_status_id"`
```

Comment:
I don't know this for sure, but asynchronous job IDs doesn't seem like the kind of thing that would be in the CDI spec. If it isn't, then this should be camelCase in JSON as `asyncStatusID`